src/App.scss

```diff
@@ -0,0 +1,32 @@
+// Source Sans Pro import
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,600;1,400;1,600&display=swap');
+// Roboto Mono import
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400;500;600&display=swap');
```

Comment:
I imagine long-term we could copy these from `cockroachdb/cockroach` and serve them ourselves, but Google Fonts is totally fine for now 😃

Reply:
I would love to eventually serve them from a CDN, but barring that I think `fonts.googleapis.com` is going to be more reliable than anything we have currently. 😄